lua/wire/client/pickcolor_browser.lua

```diff
@@ -0,0 +1,109 @@
+--[[
+	
+	ColorPicker for Expression2
+	Allows faster color picking whithin the editor.
+	
+	by NickMBR
+	Dec 2016
+	
+]]--
+
+-- Convars
+CopyWithVecString = CreateClientConVar("wire_expression2_colorwithvec", 1, true, false)
+CopyWithAlpha = CreateClientConVar("wire_expression2_copyalpha", 0, true, false)
+
+-- Follows sound_browser pattern
+-- Sets up the panel
```

Comment:
I don't think it's accurate to say that the line `local PickColorPanel = nil` is adequately documented by the above comments.